sklearn/ensemble/_forest.py

```diff
@@ -2290,6 +2441,10 @@ def __init__(self,
         self.min_impurity_split = min_impurity_split
         self.sparse_output = sparse_output
 
+    def _set_oob_permutation_importance(self, X, y):
+        raise NotImplementedError("OOB permutation importance not supported "
```

Comment:
Since we don't expose the parameter to compute the feature importance via permutation, we don't need this safeguard.